src/com/trainee/phani/Diagonal.java

```diff
@@ -0,0 +1,40 @@
+package com.trainee.phani;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Diagonal {
```

Comment:
Add java Doc explaining the purpose of the class or code

Reply:
This is not the proper way of adding a Java doc . Please follow below suggestion
/**

Content goes here

**/